### PR TITLE
Display a confirmation message for successful empty-result queries

### PR DESCRIPTION
**Pull request by a97410985**
related to https://github.com/MariaDB/mariadb_kernel/issues/15

## my ideas :
mariadb_kernel/kernel.py is reponsible for execute the code when frontend run a cell, and the function handle this is `do_execute(...)`，which would parse sql and magic command and execute it.  The following code for execute sql statements.
```python
 # part of mariadb\_kernel/kernel.py
 statements = parser.get_sql()
 for s in statements:
	 result = self.mariadb_client.run_statement(s)
	 if self.mariadb_client.iserror():
	 	self._send_message("stderr", self.mariadb\_client.error\_message())
	 	continue
 	self._update_data(result)
 	display_content = {"data": {"text/html": str(result)}, "metadata": {}}
 	if not silent:
 		self.send_response(self.iopub_socket, "display_data", display_content)

```
when error occur, `if self.mariadb_client.iserror():` will handle this. After that, if the result is empty we should display a pure text, show ''Query OK", make the user assure execution success

## demonstration(video)
https://www.youtube.com/watch?v=-3GKlLQjwGA&ab_channel=%E6%B1%9F%E6%96%B0%E7%9F%A5

### CODE REVIEW

**Review by robertbindar — APPROVED** [review]
Hey @a97410985! Great job! I think the output is perfectly good for now, we can enrich it later if needed, but looks find to me for now.

Can you please rebase and force push the PR? I will merge it once this is ready.
Also, in case you didn't read our [Contributing doc](https://github.com/MariaDB/mariadb_kernel/blob/master/CONTRIBUTING.md), I encourage you to run `black` for code formatting. For this PR, it seems GitHub actions failed to trigger, but we have a `pre-commit` job that checks if black returns any code formatting errors for each PR.

**Comment by a97410985** [comment]
sorry for not code formatting my code, I rebase and force push to my PR. Seems fine.

**Comment by robertbindar** [comment]
Hey @a97410985! No worries, we're lucky we can format so easy using this tool. Please update the PR again, sorry for this, when you rebased, you missed the `self._styled_result(result)` call I introduced before your commit.

File: mariadb_kernel/kernel.py
Comment (robertbindar) [inline mariadb_kernel/kernel.py:138]: Here, you missed the _styled_result call (see above in deleted lines).

Comment (a97410985) [reply mariadb_kernel/kernel.py:138]: I got it. 😐

Comment (robertbindar) [reply mariadb_kernel/kernel.py:138]: Actually the _styled_result call needs to be here, i.e. `"..html": self._styled_result(result_str)`, and the `display_content` dict below needs to be left alone because _styled_result acts on a HTML string directly, not a dictionary type.
It is my fault as it is hard for you to figure out what's the use case for this, we don't have a test for the bug I fixed with this styled_result function. I will add a test this evening. Thanks for the patience.

And because we've both just seen how important a test case is, can you please also add a test case for this PR in `test_mariadbclient.py`? The test should run a `use test;` statement for instance, and the expected result should be `Query OK`. I promise this is the last request before merge, thanks a lot for the patience 🙇

Comment (a97410985) [reply mariadb_kernel/kernel.py]: Maybe fix it and I add a test for empty-result queries. Unit test is important🤣. This is my first time open a pull request. Maybe I ignore something or make mistake. Thanks for a lot of patience with me😀.

**Comment by robertbindar** [comment]
Thanks a lot for the contribution @a97410985! Awesome work :-)

**Pull request merged**
